File: src/main/java/io/github/alexiscomete/lapinousecond/PlayerShop.java
package io.github.alexiscomete.lapinousecond;

import org.javacord.api.event.message.MessageCreateEvent;

public class PlayerShop extends CommandBot {
    public PlayerShop() {
        super("description", "ps", "totalDescription");
    }

    @Override
    void execute(MessageCreateEvent messageCreateEvent, String content, String[] args) {

    }
}

File: src/main/java/commands/classes/PlayerShop.kt
package commands.classes

import commands.CommandBot
import org.javacord.api.event.message.MessageCreateEvent

class PlayerShop : CommandBot(
    "Marché des joueurs",
    "playershop",
    "Vous pouvez vendre ici des items pour le prix que vous voulez (playershop sell). C'est ensuite aux autres joueurs de l'acheter (playershop buy et playershop list)"
) {
    override fun execute(messageCreateEvent: MessageCreateEvent, content: String, args: Array<String>) {}
}